### PR TITLE
Improve inventory and authorization tests

### PR DESCRIPTION
**Pull request by Exellin**

### CODE REVIEW

**Comment by Exellin** [comment]
Not sure how to reduce the lines in the inventory name block.

File: spec/models/inventory_spec.rb [hunk 1]
```diff
@@ -18,9 +18,18 @@
       expect(FactoryGirl.build(:inventory, name: Faker::Lorem.characters(256))).not_to be_valid
     end
 
-    it 'is invalid with a duplicate value' do
-      FactoryGirl.create(:inventory, name: 'duplicate')
-      expect(FactoryGirl.build(:inventory, name: 'Duplicate')).not_to be_valid
+    context 'uniqueness' do
```
Comment (pachonk) [inline spec/models/inventory_spec.rb]: More of a describe than a context. context is usually used to say "when" like "when a user is authenticated"

File: spec/features/users/sign_in_spec.rb [hunk 2]
```diff
@@ -16,6 +16,13 @@
   end
   
   scenario 'fails with invalid credentials' do
+    fill_in 'Email', with: 'wrong_email@example.com'
+    fill_in 'Password', with: 'wrong_password'
+    click_button 'Log in'
+    expect(page).to have_content('Invalid Email or password')
```
Comment (pachonk) [inline spec/features/users/sign_in_spec.rb]: Rather than expecting it TO SHOW an error, you should expect it to NOT successfully sign in.

**Comment by pachonk** [comment]
There's a broken rubocop rule currently. We should probably ignore that rule on spec/ dir in .rubocop.yml.

**Comment by Exellin** [comment]
I'm going to fix and rebase this pull request once we fix and pull in the user and inventory relationship pull request which also has the same error.

**Comment by pachonk** [comment]
@Exellin needs rebase for travis to pass.

**Comment by Exellin** [comment]
There are still a few changes I want to make on this which involve the other PR so I'm holding off for now.

**Comment by Exellin** [comment]
@pachonk updated and ready for another review.

**Pull request merged**